playbooks/roles/tower/defaults/main.yml

```diff
@@ -71,5 +72,66 @@ tower_job_template_authenticate_openshift_job_credentials: "{{ tower_credential_
 
 # Ansible Tower Hosts
 tower_host_local_name: 'local'
-tower_host_local_desc: ""
+tower_host_local_desc: ''
+
+# Ansible Tower Installation Configurations
+tower_version: '3.4.3'
+tower_install_dir: "/tmp/ansible-tower-openshift-setup-{{ tower_version }}"
+tower_archive_filename: "ansible-tower-openshift-setup-{{ tower_version }}.tar.gz"
+tower_archive_url: "https://releases.ansible.com/ansible-tower/setup_openshift/{{ tower_archive_filename }}"
+
+kubernetes_task_version: 'latest'
+kubernetes_web_version: 'latest' 
+
+kubernetes_base_path: "{{ local_base_config_path|default('/tmp') }}/{{ kubernetes_deployment_name }}-config"
+kubernetes_task_image: quay.io/integreatly/ansible-tower-container
+kubernetes_web_image: quay.io/integreatly/ansible-tower-container
+
+openshift_skip_tls_verify: true
+openshift_pg_emptydir: true
+
+dockerhub_base: ansible
+create_preload_data: True
+admin_user: admin
```

Comment:
Might be better to rename these vars to: `tower_admin_user` and `tower_admin_password`. The current names are quite broad

Reply:
Thinking about it. might be a good idea to prefix all of these vars with `tower_` to make it clear that they are specifically for Ansible Tower